dragula.js

```diff
@@ -605,4 +613,66 @@ function getCoord (coord, e) {
   return host[coord];
 }
 
+function getScrollContainer(node) {
+  if (node === null) { return null; }
+  if (node.scrollHeight > node.clientHeight) {
+    return node;
+  } else {
+    if (!/(body|html)/i.test(node.parentNode.tagName)) {
```

Comment:
I created a commit for it, e9e4e5a6eb71b9ce726ddbc17b995a822aad6110; but I'm not sure if I'm happy with this path. I'm open to suggestions.